core/src/main/java/org/apache/iceberg/util/SnapshotUtil.java

```diff
@@ -236,7 +238,7 @@ public static List<DataFile> newFiles(Long baseSnapshotId, long latestSnapshotId
         return newFiles;
       }
 
-      Iterables.addAll(newFiles, currentSnapshot.addedFiles());
+      Iterables.addAll(newFiles, currentSnapshot.addedFiles(fileIO));
```

Comment:
Don't forget to rename `fileIO` to `io` here (and throughout)

Reply:
I believe this was already present and named `fielIO`. But I can rename them if we'd like.

Reply:
Names that were already present are fine to leave in. Just don't create any new variables named `fileIO` unless you have to because they conflict with instance fields.

Reply:
Oh nevermind this wasn't already present. Renamed to `io`.